pkg/output/output.go

```diff
@@ -125,7 +127,7 @@ func MalwaresToMalwareInfo(out IOCFound) *pb.MalwareInfo {
 	stringsToMatch := make([]string, 0)
 	for i := range out.Meta {
 		if !utf8.ValidString(out.Meta[i]) && bool {
```

Comment:
Are we sure we want to use validstrings from an encoding library ? Does this mean that the input strings are also utf ? Considering that this is open source, we should expect non-utf systems too (chinese & few other languages ??). How does this work in that scenario ??